### PR TITLE
🏃 [0.5] Update dependencies to v0.17.8

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

/assign @alvaroaleman 
/milestone v0.5.x

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -24,11 +25,11 @@ require (
...
-	k8s.io/api v0.17.7
-	k8s.io/apiextensions-apiserver v0.17.7
-	k8s.io/apimachinery v0.17.7
-	k8s.io/client-go v0.17.7
-	k8s.io/utils v0.0.0-20200603063816-c1c6865ac451
+	k8s.io/api v0.17.8
+	k8s.io/apiextensions-apiserver v0.17.8
+	k8s.io/apimachinery v0.17.8
+	k8s.io/client-go v0.17.8
+	k8s.io/utils v0.0.0-20200619165400-6e3d28b6ed19
```
Comment (alvaroaleman) [inline go.mod:32]: This got downgraded?

Comment (vincepri) [reply go.mod:32]: It's newer `2020 06 03` and `2020 06 19`  in this addition

**Comment by vincepri** [comment]
@alvaroaleman 🤔 That's a little strange, they _should_ run. Going to check tomorrow in test-infra

**Comment by vincepri** [comment]
Opened github.com/kubernetes/test-infra/pull/18262 to enable them, seems we never got there. This PR should be fine though (I'm not 100% sure that they will run after we merge the test-infra one)

**Comment by vincepri** [comment]
Seems it's working :)

**Pull request merged**